### PR TITLE
feat: add harmony-staking strategy [harmony-staking]

### PR DESCRIPTION
**Pull request by MaxMustermann2**
calculates votes of validators through the total balance delegated to
them (including self delegation)

Changes proposed in this pull request:
- add harmony staked balance strategy for validator addresses

### CODE REVIEW

File: src/strategies/harmony-staking/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export const author = 'harmony-one';
+export const version = '0.0.1';
+
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
```
Comment (ChaituVR) [inline src/strategies/harmony-staking/index.ts:6]:
```suggestion
import { StaticJsonRpcProvider } from '@ethersproject/providers';
import { formatUnits } from '@ethersproject/units';
import { BigNumber } from '@ethersproject/bignumber';

export const author = 'harmony-one';
export const version = '0.0.1';

```

File: src/strategies/harmony-staking/index.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        [blockTag],
+    );
+    return Object.fromEntries(
+        Object.entries(response).map(([address, balance]) => [
+          address,
+          parseFloat(
+            formatUnits(
+                BigNumber.from('0x' + balance.toString(16)),
+                options && options.decimals ? options.decimals: 18,
+            )
```
Comment (ChaituVR) [inline src/strategies/harmony-staking/index.ts:40]: Hey @MaxMustermann2 I think `contract-call` strategy should work in this case, let me know if it doesn't work :) Thanks

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/harmony-staking/index.ts (hunk 2 above)
Comment (MaxMustermann2) [reply src/strategies/harmony-staking/index.ts:40]: Hey @ChaituVR, this is an RPC call and not a contract call that uses the EVM. I don't think `contract-call` will work (there is no ABI or contract address).

**Review by ChaituVR — APPROVED** [review]
Thanks for explaining @MaxMustermann2 :)

**Pull request merged**